core/modules/commands/savewikifolder.js

```diff
@@ -32,17 +33,23 @@ var Command = function(params,commander,callback) {
 };
 
 Command.prototype.execute = function() {
-	if(this.params.length < 1) {
+	if(typeof(this.params.folder) == "undefined") {
 		return "Missing wiki folder path";
 	}
-	var wikifoldermaker = new WikiFolderMaker(this.params[0],this.params[1],this.commander);
+	if(this.params.explodeplugins == "no" || typeof(this.params.explodeplugins) == "undefined"){
```

Comment:
usually we use `===` to avoid type-casting errors ... also for the `== "undefined"` in line 36

Reply:
ok